scripts/build/builders/bouffalolab.py

```diff
@@ -14,10 +14,20 @@
 
 import os
 import platform
+import logging
 from enum import Enum, auto
 
 from .gn import GnBuilder
 
+# Supported log levels, mapping string values required for argument
+# parsing into logging constants
+__LOG_LEVELS__ = {
```

Comment:
I don't think you need `__LOG_LEVELS__` since these are generally used for `main` arguments only